lib/stepmod/utils/converters/ol.rb

```diff
@@ -9,8 +9,9 @@ def convert(node, state = {})
           anchor = id ? "[[#{id}]]\n" : ""
           ol_count = state.fetch(:ol_count, 0) + 1
           attrs = ol_attrs(node)
-          res = "\n#{anchor}#{attrs}#{treat_children(node, state.merge(ol_count: ol_count))}"
-          res = "\n" + res if node.parent && node.parent.name == 'note'
+          res = "\n\n#{anchor}#{attrs}#{treat_children(node, state.merge(ol_count: ol_count))}\n"
+          # Why do we add this?
+          # res = "\n" + res if node.parent && node.parent.name == 'note'
```

Comment:
If a list item contains a note, it should be done like this:

```xml
<li>list item<note>my note</note></li>
```

=> 

```adoc
* list item
+
[NOTE]
--
my note
--
```

Reply:
This line ads a new line in case the list is inside the `note` tag, we even have the specific spec for it. Abou the case you described above, we already do that by default.

Reply:
Great, I don't have any further issues then.